test/passing/exceptions.ml

```diff
@@ -1,4 +1,4 @@
-exception EvalError of Error.t [@@deriving sexp]
+exception EvalError of Error.t
```

Comment:
@xclerc, your migrate-parsetree branch might need to be updated.
In 4.07, the following where equivalent:
- `exception EvalError of Error.t [@@deriving sexp]`
- `exception EvalError of Error.t [@deriving sexp]`

It's no longer the case in 4.08, the downgrade 4.08 -> 4.07 is one of the two on the floor.

Reply:
I've made the change in https://github.com/hhugo/ocaml-migrate-parsetree/commit/941337e273bcb19531be54c3dc22a95cf1a1c1e4